### PR TITLE
Hide comments section when there are no comments and the comment form is disabled.

### PR DESCRIPTION
**Pull request by ghost**
Addresses https://github.com/backdrop/backdrop-issues/issues/2926

### CODE REVIEW

File: core/modules/node/templates/node.tpl.php [hunk 1]
```diff
@@ -109,7 +109,7 @@
 
   <?php print render($content['links']); ?>
 
-  <?php if ($page && isset($comments['comments'])): ?>
+  <?php if ($page && (isset($comments['comments']) || isset($comments['comment_form']))): ?>
```
Comment (jenlampton) [inline core/modules/node/templates/node.tpl.php:112]: Do we also need to add this change to Bartik's version of node.tpl.php?

**Review by jenlampton — COMMENTED** [review]
Looks great!  I had only a single thought...

File: core/modules/node/templates/node.tpl.php (hunk 1 above)
Comment (jenlampton) [reply core/modules/node/templates/node.tpl.php:112]: Looks like it:
```php
  <?php if ($page && isset($comments['comments'])): ?>
```

**Comment by ghost** [comment]
Thanks @jenlampton, done!

**Comment by quicksketch** [comment]
In testing this locally, I still get the `<section class="comments">` wrapper, even when there are no comments and the form is disabled. Looks like this isn't doing what we hope to accomplish here.

**Comment by quicksketch** [comment]
I filed a new PR at https://github.com/backdrop/backdrop/pull/2160 as an alternative to this approach.

**Comment by jenlampton** [comment]
I prefer the approach in https://github.com/backdrop/backdrop/pull/2160, the template looks much cleaner, and I'm far less confused :) Closing this PR in favor of that one.

**Pull request closed without merging**